src/cli.rs

```diff
@@ -66,8 +66,8 @@ impl Cli {
 }
 
 impl Behavior for Cli {
-  fn startup_snapshot(&mut self) -> Option<deno_buf> {
-    self.init.snapshot.take()
+  fn startup_data(&mut self) -> Option<StartupData> {
```

Comment:
Please add a docstring here
```
/// Allows the behavior to define either a snapshot or a JavaScript file to be run on startup.
```

Reply:
Is it ok if I put the documentation in the `Behavior` trait definition instead? I figure that is where this information belongs.

Reply:
Yes - thanks!